plugins/modules/files/iso_create.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Ansible Project
+# Copyright: (c) 2020, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: iso_create
+short_description: Generate ISO file with specified files or folders
+description:
+    - This module is used to generate ISO file with specified path of files.
+version_added: '2.10'
```

Comment:
```suggestion
```
to avoid that sanity error, please, don't worry about version_added. we'll add it later, after this module is merged